hvplot/tests/testutil.py

```diff
@@ -361,3 +369,22 @@ def test_instantiate_crs_str_kwargs():
     assert isinstance(crs, ccrs.PlateCarree)
     assert isinstance(crs.globe, ccrs.Globe)
     assert crs.globe.datum == 'WGS84'
+
+
+@pytest.mark.skipif(
+    spatialpandas is None or dask is None, reason='spatialpandas or dask is not available'
+)
+def test_is_geodataframe_spatialpandas_dask():
+    import dask.dataframe as dd
+    import spatialpandas as spd
```

Comment:
One other thought are these can be moved up, and the pytest.mark.skipif can check if dd or spd is None.
```suggestion
```

Reply:
OK. Done